Sources/SQLiteNIO/SQLiteCustomFunction.swift

```diff
@@ -358,45 +333,47 @@ extension SQLiteCustomFunction {
 ///
 /// For example:
 ///
-///     struct MySum : DatabaseAggregate {
+///     struct MySum: DatabaseAggregate {
 ///         var sum: Int = 0
 ///
-///         mutating func step(_ dbValues: [DatabaseValue]) {
-///             if let int = Int.fromDatabaseValue(dbValues[0]) {
+///         mutating func step(_ dbValues: [SQLiteData]) {
+///             if let int = dbValues[0].integer {
 ///                 sum += int
 ///             }
 ///         }
 ///
-///         func finalize() -> DatabaseValueConvertible? {
-///             return sum
+///         func finalize() -> (any SQLiteDataConvertible)? {
+///             sum
 ///         }
 ///     }
 ///
 ///     let connection: SQLiteConnection = ...
 ///     let fn = SQLiteCustomFunction("mysum", argumentCount: 1, aggregate: MySum.self)
-///     try connection.install(customFunction: fn).wait()
-///     try connection.query("CREATE TABLE test(i)").wait()
-///     try connection.query("INSERT INTO test(i) VALUES (1)").wait()
-///     try connection.query("INSERT INTO test(i) VALUES (2)").wait()
-///     let sum: Int = try connection.query("SELECT mysum(i) FROM test")!.wait()
+///     try await connection.install(customFunction: fn).get()
+///     try await connection.query("CREATE TABLE test(i)").get()
+///     try await connection.query("INSERT INTO test(i) VALUES (1)").get()
+///     try await connection.query("INSERT INTO test(i) VALUES (2)").get()
+///     let sum = (try await connection.query("SELECT mysum(i) FROM test").get().first?.columns.first?.integer)!
 public protocol SQLiteCustomAggregate {
-    /// Creates an aggregate.
+    /// Create an aggregate.
     init()
 
     /// This method is called at each step of the aggregation.
     ///
     /// The dbValues argument contains as many values as given to the SQL
     /// aggregate function.
     ///
-    ///    -- One value
-    ///    SELECT maxLength(name) FROM player
+    ///     -- One value
+    ///     SELECT maxLength(name) FROM player
     ///
-    ///    -- Two values
-    ///    SELECT maxFullNameLength(firstName, lastName) FROM player
+    ///     -- Two values
+    ///     SELECT maxFullNameLength(firstName, lastName) FROM player
     ///
-    /// This method is never called after the finalize() method has been called.
+    /// This method is never called after the ``finalize()`` method has been called.
     mutating func step(_ values: [SQLiteData]) throws
 
-    /// Returns the final result
-    func finalize() throws -> SQLiteDataConvertible?
+    /// Return the final result. Called only once, at the end of the aggregation.
+    func finalize() throws -> (any SQLiteDataConvertible)?
 }
+
+extension SQLiteCustomFunction: @unchecked Sendable {}
```

Comment:
Again, do we need to disable checking for the whole type? It's only `eTextRep` that's requiring this right?

Reply:
Turns out _nothing_ requires it once I added the missing `@Sendable` annotations. I was able to mark the type `Sendable` without disabling checking.